rocky/rocky/views/mixins.py

```diff
@@ -380,23 +379,21 @@ def get_children_input_oois(children_reports: list[Report]) -> set[str]:
         return {input_ooi for child_report in children_reports for input_ooi in child_report.input_oois}
 
     @staticmethod
-    def report_type_summary(reports: list[Report]) -> dict[str, int]:
+    def report_type_summary(reports: list[Report]) -> dict[str, int] | None:
         """
         Calculates per report type how many objects it consumed.
         """
 
         summary: dict[str, int] = {}
         report_types: set[str] = {report.report_type for report in reports}
 
-        for report_type in report_types:
-            summary[report_type] = len(
-                {
-                    itertools.chain.from_iterable(
-                        [report.input_oois for report in reports if report_type == report.report_type]
-                    )
-                }
-            )
-
+        if report_types:
+            for report_type in sorted(list(report_types)):
+                count_oois = 0
+                for report in reports:
+                    if report_type == report.report_type:
+                        count_oois += len(report.input_oois)
+                summary[report_type] = count_oois
```

Comment:
You could've also changed these two statements:
- `summary[report_type] = len` to `summary[report_type] = sum`
- `[report.input_oois for ...]` to `[len(report.input_oois) for ...]`